onnxruntime/core/util/thread_utils.cc

```diff
@@ -9,7 +10,74 @@
 #include "core/session/ort_apis.h"
 
 namespace onnxruntime {
+
+#ifdef _WIN32
+GroupAffinities GetGroupAffinities() {
```

Comment:
>GetGroupAffinities [](http://example.com/codeflow?start=16&length=18)

Rename to GetDefault...
And make sure when any system call fails, return std::hardware_concurrency//2